app/src/main/res/layout/fragment_events.xml

```diff
@@ -87,7 +87,13 @@
                     <FrameLayout
                         android:layout_width="match_parent"
                         android:layout_height="match_parent">
-
+                        <TextView
+                            android:id="@+id/noEventsMessage"
+                            android:layout_width="match_parent"
+                            android:layout_height="450dp"
```

Comment:
Please define all dimensions in dimension resources file.

Reply:
I will fix it now

Reply:
I fixed it, please review